hail_search/test_search.py

```diff
@@ -117,6 +128,29 @@ async def test_multi_project_search(self):
             sample_data=MULTI_PROJECT_SAMPLE_DATA,
         )
 
+    async def test_inheritance_filter(self):
+        await self._assert_expected_search(
+            [VARIANT1, VARIANT2, MULTI_FAMILY_VARIANT, VARIANT4], inheritance_mode='any_affected', omit_sample_type='SV_WES',
```

Comment:
You may want to add a test for both inheritance_filter and inheritance_mode are undefined.

Reply:
thats literally every other test in the file